src/main/java/de/retest/recheck/ignore/PersistentFilter.java

```diff
@@ -0,0 +1,53 @@
+package de.retest.recheck.ignore;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.diff.AttributeDifference;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+@ToString( onlyExplicitlyIncluded = true )
+@AllArgsConstructor
+public class PersistentFilter implements Filter {
+
+	@Getter
+	private final Path path;
+
+	@ToString.Include
+	@Getter
+	// Delegate
+	private final Filter filter;
+
+	public static Stream<PersistentFilter> wrap( final Path path, final Stream<Filter> filter ) {
+		return filter.map( f -> new PersistentFilter( path, f ) );
+	}
+
+	public static List<Filter> unwrap( final List<? extends Filter> input ) {
+		return input.stream().map( f -> f instanceof PersistentFilter ? ((PersistentFilter) f).getFilter() : f )
+				.collect( Collectors.toList() );
+	}
+
+	@Override
+	public boolean matches( final Element element ) {
+		// Delegate
```

Comment:
Why the comments? It should be obvious. But in case it is not, why not just rename the `filter` to `delegate`

Reply:
Because then the generated getter and setter methods are also renamed, and that I don't want.